bindings/azure/storagequeues/storagequeues.go

```diff
@@ -119,37 +136,41 @@ func (d *AzureQueueHelper) Write(ctx context.Context, data []byte, ttl *time.Dur
 		ttlToUse := defaultTTL
 		ttl = &ttlToUse
 	}
-	_, err = messagesURL.Enqueue(ctx, s, time.Second*0, *ttl)
+	_, err = d.queueClient.EnqueueMessage(ctx, s, &azqueue.EnqueueMessageOptions{
+		TimeToLive: ttlSeconds,
+	})
 
 	return err
 }
 
 func (d *AzureQueueHelper) Read(ctx context.Context, consumer *consumer) error {
-	messagesURL := d.queueURL.NewMessagesURL()
-	res, err := messagesURL.Dequeue(ctx, 1, d.visibilityTimeout)
+	res, err := d.queueClient.DequeueMessages(ctx, &azqueue.DequeueMessagesOptions{
+		NumberOfMessages:  ptr.Of(int32(1)),
+		VisibilityTimeout: ptr.Of(int32(d.visibilityTimeout.Seconds())),
+	})
 	if err != nil {
 		return err
 	}
-	if res.NumMessages() == 0 {
+	if len(res.Messages) == 0 {
 		// Queue was empty so back off by 10 seconds before trying again
 		select {
 		case <-time.After(10 * time.Second):
```

Comment:
Should this be configurable?

Reply:
Maybe - but this PR is just the SDK upgrade

Reply:
A separate issue/PR is fine!